packages/@aws-cdk/aws-apigateway/test/test.method.ts

```diff
@@ -304,6 +304,63 @@ export = {
     test.done();
   },
 
+  'methodResponse set one or more method responses via options'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const api = new apigateway.RestApi(stack, 'test-api', { deploy: false });
+
+    // WHEN
+    new apigateway.Method(stack, 'method-man', {
+      httpMethod: 'GET',
+      resource: api.root,
+      options: {
+        methodResponses: [{
+            statusCode: '200'
+          }, {
+            statusCode: "400",
+            responseParameters: {
+              'method.response.header.killerbees': false
+            }
+          }, {
+            statusCode: "500",
+            responseParameters: {
+              'method.response.header.errthing': true
+            },
+            responseModels: {
```

Comment:
I am wondering if a bunch of static properties on Model to make it “enum-like” would make a nicer and more discoverable API here: “Model.Empty” and “Model.Error”

Reply:
I explored adding static properties to a stubbed Model class, but I get an error that the build wants me to have Model implement `cdk.Construct`, so I backed away for now. Perhaps there's a way to get the build to temporarily ignore that requirement?